src/current/v24.3/architecture/replication-layer.md

```diff
@@ -146,6 +146,35 @@ A table's meta and system ranges (detailed in the [distribution layer]({% link {
 
 However, unlike table data, system ranges cannot use epoch-based leases because that would create a circular dependency: system ranges are already being used to implement epoch-based leases for table data. Therefore, system ranges use expiration-based leases instead. Expiration-based leases expire at a particular timestamp (typically after a few seconds). However, as long as a node continues proposing Raft commands, it continues to extend the expiration of its leases. If it doesn't, the next node containing a replica of the range that tries to read from or write to the range will become the leaseholder.
 
+#### Leader leases
+
+{% include_cached new-in.html version="v24.3" %}
+
+{% include feature-phases/preview.md %}
+
+{% include {{ page.version.version }}/leader-leases-intro.md %}
+
+CockroachDB offers an improved leasing system rebuilt atop a stronger form of [Raft](#raft) leadership that ensures that the Raft leader is **always** the range's leaseholder. This new type of lease is called a _Leader lease_, and supersedes [epoch-based leases](#epoch-based-leases-table-data) and [expiration-based leases](#expiration-based-leases-meta-and-system-ranges) leases while combining the performance of the former with the resilience of the latter.
```

Comment:
The {include} and the first line of the actual doc are the same, so in the preview it shows up twice

Reply:
oof thanks Noman, fixed that in latest commit `Update with nhamlani101 feedback (1)`